src/demo/packet/datatable.rs

```diff
@@ -58,6 +58,16 @@ impl PartialEq<u16> for ClassId {
     }
 }
 
+impl ClassId {
```

Comment:
Required to make matching on `ClassId` available in the analyzer (due to the value being a private member of the struct)

Reply:
Turns out this wasn't needed - class IDs should be looked up from the parser state instead of being hardcoded